public/theme/extensions-2021/css/bulma.css

```diff
@@ -0,0 +1,3 @@
atio,.image.is-square img,.is-overlay,.modal,.modal-background{bottom:0;left:0;position:absolute;right:0;top:0}.boltform button,.boltform input,.boltform select select,.boltform textarea,.button,.file-cta,.file-name,.input,.pagination-ellipsis,.pagination-link,.pagination-next,.pagination-previous,.select select,.textarea{-moz-appearance:none;-webkit-appearance:none;align-items:center;border:1px solid transparent;border-radius:4px;box-shadow:none;display:inline-flex;font-size:1rem;height:2.5em;justify-content:flex-start;line-height:1.5;padding:calc(.5em - 1px) calc(.75em - 1px);position:relative;vertical-align:top}.boltform button.is-active,.boltform button.is-focused,.boltform button:active,.boltform button:focus,.boltform input.is-active,.boltform input.is-focused,.boltform input:active,.boltform input:focus,.boltform select select.is-active,.boltform select select.is-focused,.boltform select select:active,.boltform select select:focus,.boltform textarea.is-active,.boltform textarea.is-focused,.boltform textarea:active,.boltform textarea:focus,.button:active,.button:focus,.file-cta:active,.file-cta:focus,.file-name:active,.file-name:focus,.input:active,.input:focus,.is-active.button,.is-active.file-cta,.is-active.file-name,.is-active.input,.is-active.pagination-ellipsis,.is-active.pagination-link,.is-active.pagination-next,.is-active.pagination-previous,.is-active.textarea,.is-focused.button,.is-focused.file-cta,.is-focused.file-name,.is-focused.input,.is-focused.pagination-ellipsis,.is-focused.pagination-link,.is-focused.pagination-next,.is-focused.pagination-previous,.is-focused.textarea,.pagination-ellipsis:active,.pagination-ellipsis:focus,.pagination-link:active,.pagination-link:focus,.pagination-next:active,.pagination-next:focus,.pagination-previous:active,.pagination-previous:focus,.select select.is-active,.select select.is-focused,.select select:active,.select select:focus,.textarea:active,.textarea:focus{outline:none}.boltform button[disabled],.boltform fieldset[disabled] button,.boltform fieldset[disabled] input,.boltform fieldset[disabled] textarea,.boltform input[disabled],.boltform select fieldset[disabled] select,.boltform select select[disabled],.boltform textarea[disabled],.button[disabled],.file-cta[disabled],.file-name[disabled],.input[disabled],.pagination-ellipsis[disabled],.pagination-link[disabled],.pagination-next[disabled],.pagination-previous[disabled],.select fieldset[disabled] select,.select select[disabled],.textarea[disabled],fieldset[disabled] .boltform button,fieldset[disabled] .boltform input,fieldset[disabled] .boltform select select,fieldset[disabled] .boltform textarea,fieldset[disabled] .button,fieldset[disabled] .file-cta,fieldset[disabled] .file-name,fieldset[disabled] .input,fieldset[disabled] .pagination-ellipsis,fieldset[disabled] .pagination-link,fieldset[disabled] .pagination-next,fieldset[disabled] .pagination-previous,fieldset[disabled] .select select,fieldset[disabled] .textarea{cursor:not-allowed}
```

Comment:
I don't think we use Bulma in this project, so is this file needed?

Reply:
No, deleted.